plugins/plugin-build-script/plugin.js

```diff
@@ -29,11 +29,9 @@ function buildScriptPlugin(_, {input, output, cmd}) {
       if (exitCode !== 0) {
         throw new Error(stderr || stdout);
       }
-      // If the plugin output tp stderr, how it to the user.
-      if (stderr) {
-        throw new Error(stderr);
-      }
-      return {[output[0]]: stdout};
+
+      const output = [stderr, stdout].filter((value) => !!value).join('\n\n');
+      return {[output[0]]: output};
```

Comment:
+1 for not throwing if `stderr` output exists, this use-case makes sense. But, stderr is almost always error/log messages and never relevant "build output". Instead, lets jsut `console.warn`:

```js
if (stderr) {
  console.warn(stderr);
}
return {[output[0]]: stdout};
```

Reply:
good point.